### PR TITLE
2023.1: Improvements to Ceph docs & more examples

### PR DESCRIPTION
**Pull request by markgoddard**
- **docs: Minor Ceph changes**
- **docs: Include encryption at rest in Ceph OSD example**
- **docs: Add an example of Ceph messenger v2 encryption at rest**
- **multinode: Revert to default Ceph release & tag**

### CODE REVIEW

File: doc/source/configuration/cephadm.rst [hunk 1]
```diff
@@ -107,7 +107,7 @@ OSD specification
 ~~~~~~~~~~~~~~~~~
 
 The following example is a basic OSD spec that adds OSDs for all
-available disks:
+available disks with encryption at rest:
```
Comment (MoteHue) [inline doc/source/configuration/cephadm.rst:110]: Is this definitely something we want to suggest as default? Mainly wondering if there's any performance impact.

Comment (markgoddard) [reply doc/source/configuration/cephadm.rst:110]: I think it's fairly standard in our deployments these days. No doubt there will be some performance impact, but unsure how much.

**Comment by Alex-Welsh** [comment]
Still draft?

**Comment by markgoddard** [comment]
> Still draft?

Waiting for Jack to test the secure bit.

**Review by jovial — APPROVED** [review]

**Pull request merged**